setup.py

```diff
@@ -22,11 +22,24 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-from distutils.core import setup
-from server import __version__
+from setuptools import setup, find_packages
+from facts.server import __version__
 setup(
   name='fiware-facts',
-  packages=['fiware-facts'],  # this must be the same as the name above
+  packages=find_packages(exclude=['tests*']),
+  install_requires=["redis==2.9.1",
```

Comment:
I recommend you to read all dependencies from `requirements.txt`, instead of having this duplicity.
You can use this code:

      from pip.req import parse_requirements
      # Get requirements list from requirements.txt file
      # > parse_requirements() returns generator of pip.req.InstallRequirement objects
      install_reqs = parse_requirements(REQUIREMENTS_FILE, session=False)
      # > requirements_list is a list of requirement; e.g. ['requests==2.6.0', 'Fabric==1.8.3']
      requirements_list = [str(ir.req) for ir in install_reqs]